PocketKit/Sources/PocketKit/MyList/CompactMyListContainerCoordinator.swift

```diff
@@ -167,19 +171,32 @@ class CompactMyListContainerCoordinator: NSObject {
 }
 
 extension CompactMyListContainerCoordinator: UINavigationControllerDelegate {
-    func navigationController(_ navigationController: UINavigationController, didShow viewController: UIViewController, animated: Bool) {
+    func navigationController(_ navigationController: UINavigationController, willShow viewController: UIViewController, animated: Bool) {
```

Comment:
changed from `didShow` to `willShow` so that we are deselecting faster and matches the experience of the safari view; not sure if there was a specific reason we used `didShow` previously